scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -227,7 +224,11 @@ class MicrobeEditor{
 
     private void _addOrganelle(PlacedOrganelle@ organelle)
     {
-        EditorAction@ action = EditorAction(organelle.organelle.mpCost,
+
+        if((organelle.organelle.name == "nucleus" && !nucleusIsPresent) || (nucleusIsPresent && (organelle.organelle.name == "mitochondrion" || organelle.organelle.name == "chloroplast" || organelle.organelle.name == "nitrogenfixingplastid" || organelle.organelle.name == "chemoplast")) || (organelle.organelle.name == "cytoplasm" || organelle.organelle.name == "flagellum" || organelle.organelle.name == "chromatophors" || organelle.organelle.name == "metabolosome" || organelle.organelle.name == "oxytoxy" || organelle.organelle.name == "vacuole" ))
```

Comment:
This line is too long

Reply:
you mean as text formatting or you mean it's bad idea solve problem in that way?

Reply:
Now that I read the whole line, it obviously should be a configuration property on the organelles if they need a nucleus, instead of this monstrous line here.